bundles/net.tourbook.chart/src/net/tourbook/chart/ComputeChartValue.java

```diff
@@ -30,5 +30,8 @@
 	 */
 	public int				breakTime;
 
+   //CUSTOM TRACK Id
+   public String custTrackId;
```

Comment:
I personally am not a fan of abbreviations....
For example, is it cust**omer** track id ?
`customTrackId `would make more sense and then you wouldn't have to repeat yourself in the comment above as it would be obvious now what the variable contians.

Reply:
@FJBDev done 👍